VoxelSniperCore/src/main/java/com/github/kevindagame/snipe/Sniper.java

```diff
@@ -129,7 +129,6 @@ private boolean handleRightClick(IBlock clickedBlock, SnipeTool sniperTool, Snip
         }
 
         var success = sniperTool.getCurrentBrush().perform(snipeAction, snipeData, targetBlock, lastBlock);
-        new PlayerSnipeEvent(this.getPlayer(), sniperTool.getCurrentBrush(), success).callEvent();
```

Comment:
Was this call moved? The event doesn't seem to be called at al right now

Reply:
That is because it's in the wrong place right now, and the right place is on another unmerged branch

Reply:
ok fine ;)